dali/c_api/c_api_test.cc

```diff
@@ -610,6 +636,100 @@ TYPED_TEST(CApiTest, UseCopyKernel) {
   }
 }
 
+
+TYPED_TEST(CApiTest, NoCopyForce) {
```

Comment:
```suggestion
TYPED_TEST(CApiTest, NoCopyForceFail) {
```

Reply:
Done something similar. I think we can start with Force...